### PR TITLE
Hibernate work #1

### PR DESCRIPTION
**Pull request by grom4er**

### CODE REVIEW

File: src/main/java/cinema/Application.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            = (MovieService) injector.getInstance(MovieService.class);
+
+    public static void main(String[] args) {
+        movieService.getAll().forEach(System.out::println);
+        Movie movie = new Movie();
+        movie.setTitle("Fruit-Shop");
+        movie = movieService.add(movie);
+
+        movieService.getAll().forEach(System.out::println);
+
```
Comment (Max-Pochepets) [inline src/main/java/cinema/Application.java:20]:
```suggestion
```

**Review by Max-Pochepets — APPROVED** [review]
make sure all your files have empty lines in the end

File: src/main/java/cinema/Application.java (hunk 1 above)
Comment (grom4er) [reply src/main/java/cinema/Application.java:20]: Это я с тревисом игрался!

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+language: java
+jdk:
+  - openjdk11
+script:
+  - mvn clean install
+cache:
+  directories:
+    - $HOME/.m2
```
Comment (Sofasmile) [inline .travis.yml:8]: let's add empty line at the end of the file

File: src/main/java/cinema/dao/MovieDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(movie);
+            transaction.commit();
+            return movie;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't insert movie entity", e);
```
Comment (Sofasmile) [inline src/main/java/cinema/dao/MovieDaoImpl.java:28]:
```suggestion
            throw new DataProcessingException("Can't insert movie entity " + movie, e);
```

File: src/main/resources/hibernate.cfg.xml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+        <property name = "hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
+        <property name = "hibernate.connection.driver_class">com.mysql.jdbc.Driver</property>
+        <property name = "hibernate.connection.url">jdbc:mysql://localhost/cinema?serverTimezone=UTC</property>
+        <property name = "hibernate.connection.username">root</property>
+        <property name = "hibernate.connection.password">ssdd</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+        <mapping class="cinema.model.Movie"/>
+    </session-factory>
+</hibernate-configuration>
```
Comment (Sofasmile) [inline src/main/resources/hibernate.cfg.xml:15]: empty line

File: travis.yml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+language: java
```
Comment (Sofasmile) [inline travis.yml:1]: do we need two travis.yml file?

**Review by Sofasmile — CHANGES_REQUESTED** [review]

**Comment by Sofasmile** [comment]
firstly you should fix the conflict

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**